charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +117,68 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: "docker.io/openstorage/csi-provisioner:v1.6.1-1"
```

Comment:
Some of the CSI sidecar images depend on the k8s version. See the following code:
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/csi.go#L243

We should do one of the following:
1. Add logic to pick the provisioner, snapshotter, and snapshot-controller versions as we do in px-installer
2. Add support only for a k3s/k8s version range. This will need to documented as well

The first option is more ideal

Reply:
resolved